scripts/west_commands/export.py

```diff
@@ -39,38 +42,52 @@ def do_add_parser(self, parser_adder):
         return parser
 
     def do_run(self, args, unknown_args):
-        zephyr_config_package_path = PurePath(__file__).parents[2] \
-            / 'share' / 'zephyr-package' / 'cmake'
-
-        cmake_args = ['-S', f'{zephyr_config_package_path}',
-                      '-B', f'{zephyr_config_package_path}']
-        lines = run_cmake(cmake_args, capture_output=True)
-
-        # Let's clean up, as Zephyr has now been exported, and we no longer
-        # need the generated files.
-        cmake_args = ['--build', f'{zephyr_config_package_path}',
-                      '--target', 'pristine']
-        run_cmake(cmake_args, capture_output=True)
-
-        # Let's ignore the normal CMake printing and instead only print
-        # the important information.
-        msg = [line for line in lines if not line.startswith('-- ')]
-        print('\n'.join(msg))
-
-        zephyr_unittest_config_package_path = PurePath(__file__).parents[2] \
-            / 'share' / 'zephyrunittest-package' / 'cmake'
-
-        cmake_args = ['-S', f'{zephyr_unittest_config_package_path}',
-                      '-B', f'{zephyr_unittest_config_package_path}']
-        lines = run_cmake(cmake_args, capture_output=True)
-
-        # Let's clean up, as Zephyr has now been exported, and we no longer
-        # need the generated files.
-        cmake_args = ['--build', f'{zephyr_unittest_config_package_path}',
-                      '--target', 'pristine']
-        run_cmake(cmake_args, capture_output=True)
-
-        # Let's ignore the normal CMake printing and instead only print
-        # the important information.
+        # The 'share' subdirectory of the top level zephyr repository.
+        share = Path(__file__).parents[2] / 'share'
+
+        if which('ninja') is not None:
```

Comment:
@tejlmand requested changes here: https://github.com/zephyrproject-rtos/zephyr/pull/23977#discussion_r402954647

Reply:
Thanks.
Made a small proposal, so we can get rid of the build command completely, as we basically don't need it.
And when calling `cmake -P pristine.cmake` no build command is involved.